content/data-reporting/reports-api/index.mdx

```diff
@@ -200,6 +201,82 @@ group_by=hour" \
 
 <Text>
 
+# Chat Missed
+
+This reports shows you the number of chats with no messages sent by agents (except the first welcome message that always occur in conversations).
```

Comment:
```suggestion
This report shows you the number of chats with no messages sent by agents (except the first welcome message that always occurs in conversations).
```

Reply:
fixed in https://github.com/livechat/livechat-public-docs/pull/572/commits/72db3c1bd3011dd7ce571afc3f608468d0a8b343